pkgs/development/python-modules/libdebug/default.nix

```
@@ -0,0 +1,72 @@
+{
+  lib,
+  scikit-build-core,
+  cmake,
+  ninja,
+  pkg-config,
+  zstd,
+  zlib,
+  elfutils,
+  libdwarf,
+  libiberty,
+  fetchFromGitHub,
+  psutil,
+  pyelftools,
+  requests,
+  prompt-toolkit,
+  nanobind,
+  typing-extensions,
+  python,
+}:
+
+python.pkgs.buildPythonPackage rec {
+  pname = "libdebug";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "libdebug";
+    repo = pname;
```

We prefer avoiding this pattern, and hardcode the repository name directly 
- #277994
```suggestion
    repo = "libdebug";
```